templates/bridgeserver2-common.yaml

```diff
@@ -37,6 +37,18 @@ Resources:
             DeleteSourceFromS3: true
             Enabled: true
             MaxAgeInDays: 90
+  DBSubnetGroup:
+    Type: AWS::RDS::DBSubnetGroup
+    Properties:
+      DBSubnetGroupDescription: 'Subnet for Server2 VPC'
+      DBSubnetGroupName: 'bridgeserver2-rds-subnet'
+      SubnetIds:
+        - !ImportValue us-east-1-BridgeServer2-vpc-PublicSubnet
```

Comment:
You set the DBInstance param `PubliclyAccessible: false` so shouldn't this contain only private subnets?  is there any reason to put them in public subnets?

Reply:
The public subnets have been removed.